vendors/twtg/profiles/twtg_RFGroup4_RP2-1.0.4_classA.yaml

```
@@ -0,0 +1,101 @@
+# ID that represents the profile of your device model.
+# SHALL Follow this convention: <vendorID> (lowercase) + '_' + <Profile Group> + '_' <LoRaWAN L2 & PHY versions> + '_class' + <LoRaWAN class A, B or C>
+# Follow the information below to know the profile group to use in the ID
+#    Profile Group
+#    RFGroup1:  eu863-870, kr920-923, in865-867, ru864-870
+#    RFGroup2:  us902-928, cn470-510, au915-928 (only for LoRaWAN 1.0.2revB or below)
+#    RFGroup3:   au915-928 (starting from LoRaWAN 1.0.2revC)
+#    RFGroup4: as923
+# Please do not change the existing IDs (Some may follow the old format)
+id: twtg_RFGroup4_RP2-1.0.4_classA
+# LoRaWAN class of the Device: Possible values: [ A, B, C ] (Should not have several values)
+# 'A': Class A (Bi-directional end-devices with downlink listening only after uplink transmission)
+# 'B': Class B (Bi-directional end-devices with downlink listening on predefined synchronized pingslots)
+# 'C': Class C (Bi-directional end-devices with permanent downlink listening)
+loRaWANClass: A
+# Comma-separated list of regional profiles supported by this device. Possible values are eu868, us915, as923, au915, eu433, in865, kr920, ru864, cn470.
+ISMbands: as923
+# Typical mobility profile of the device. Possible values are 'near_static' (also valid for static devices), 'walking_speed', 'vehicular_speed'
+# or 'random' (not known, changes over time).
+# The motion indication property will not be taken into consideration here if it is overwritten in the model.yaml
+motionIndicator: RANDOM
```

Why not the same as the other profiles?
```suggestion
motionIndicator: NEAR_STATIC
```

---

Fixed